vite.config.mjs

```diff
@@ -168,6 +166,10 @@ export default defineConfig(async ({ mode }) => {
           find: '@workers',
           replacement: workersSjs,
         },
+        {
+          find: '@odbRestURL',
+          replacement: 'hello',
```

Comment:
I assume this needs to be removed before merging?

Reply:
Yeah, I was testing things here but ended up not going that way.